File: src/LC674.go
package main

func findLengthOfLCIS(nums []int) int {
	if len(nums) == 0 {
		return 0
	}
	start := 0
	count := 1
	for i := 1; i < len(nums); i++ {
		if nums[i] <= nums[i-1] {
			start = i
		} else {
			if i-start+1 > count {
				//fmt.Printf("%d %d\n", i, start)
				count = i - start + 1
			}
		}
	}
	return count
}

File: src/LC674.java
class Solution {
    public int findLengthOfLCIS(int[] nums) {
        if(nums.length==0)
            return 0;
        int count = 1;
        int max = 1;
        int pre = nums[0];
        for (int i = 1; i < nums.length; i++) {
            if (nums[i] > pre) {
                count++;
                max = Math.max(count, max);

            } else {
                count = 1;

            }
            pre = nums[i];
        }
        return max;
    }
}